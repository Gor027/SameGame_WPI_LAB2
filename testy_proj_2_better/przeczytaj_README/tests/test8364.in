RR  1R11 1 R1 11R   RR 1R11
11 111R  1 1R 
R1111 R1R   R1 1RRR1  1R1 RR1RR 1RRR
R1RRR R 11111  1RR R1   11R 1R11R  11RR1 1RR1R1 1R1  R1111R 1 1 1R1 R11 R1 1RR
 R 1R1 1 1   1R RR  11RRR1R11R 11R R 1  1111 1 1  1RRR111RRR1111   1111   R
1R
R 11R RR1RR11   1 1RRR1 R1R1RR1RR  R1111 1RRRR 1R1     1
  RRR1R  111  RR  1   1RR1 R11 1 R1 R  RRR    R
 R11RR 1 
