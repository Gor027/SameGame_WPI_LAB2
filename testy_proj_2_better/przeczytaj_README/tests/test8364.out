         1              R
   1     1 R1   R       111 1R
 R11   1 1 1R  1R   R 1111RR1R 11  R     1
R1R111 1R111R 11RRR11RRRR1R11RR1RR 11R 1 1RR R         1
111111 11RR111R1RR1R111RR1R1RR1R1RRR11R1111111111R1  R1111R 1 1     R11
11111111111111RR1R1RRR1RR1RR1111RR1RR11RRRRRRRR1R11RRR111RRR11111R11111R1R1RR
